tools/smoother_benchmarking/smoother_benchmark_bringup.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2022 Samsung Research America
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess, IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.actions import Node
+from ament_index_python.packages import get_package_share_directory
+
+def generate_launch_description():
+    nav2_bringup_dir = get_package_share_directory('nav2_bringup')
+    python_commander_dir = get_package_share_directory('nav2_simple_commander')
+
+    map_yaml_file = os.path.join(os.getcwd(), 'maps', 'smoothers_world.yaml')
+
+    # start the simulation
+    start_gazebo_server_cmd = ExecuteProcess(
```

Comment:
Shouldn't you just need the planner/smoother servers? I don't think you should need gazebo, no?

Reply:
If you look at the planner benchmark launch file,  I think this would be the same + the smoother server, no?

Reply:
[Fixed](https://github.com/ros-planning/navigation2/pull/3236/files#diff-6858250f29552fae42d8964a159abe07361c54244c44a0f848dec64193818c7a). The latest `smoother_benchmark_bringup.py` file was derived from `smoother_benchmark_bringup.py` with `smoother_server` added.